### PR TITLE
More Delectable Selection Pane

### PR DESCRIPTION
**Pull request by JrTRinny**
did what I left off from #6205, adds `onConfigureTileTapped` to command center and make mass driver and payload propulsion tower link configurations deselectable, but only when there's no linked block (basically double-tap max to deselect the config)

https://user-images.githubusercontent.com/85090668/140764159-3f6b4b1f-90a7-4ad4-a4a0-bb2c96f260c2.mp4

oh, and also fixed some code I found.

---

If your pull request is **not** translation or serverlist-related, read the list of requirements below and check each box:

- [X] I have read the [contribution guidelines](https://github.com/Anuken/Mindustry/blob/master/CONTRIBUTING.md).
- [X] I have ensured that my code compiles, if applicable.
- [X] I have ensured that any new features in this PR function correctly in-game, if applicable.

### CODE REVIEW

File: core/src/mindustry/world/blocks/storage/CoreBlock.java [hunk 1]
```diff
@@ -53,7 +53,6 @@ public CoreBlock(String name){
 
         //support everything
         envEnabled = Env.any;
-        drawDisabled = false;
```
Comment (Anuken) [inline core/src/mindustry/world/blocks/storage/CoreBlock.java:56]: why was this changed?

Comment (JrTRinny) [reply core/src/mindustry/world/blocks/storage/CoreBlock.java:56]: oh it is duplicated, there is already another one a few lines above
